.trivyignore

```diff
@@ -1,11 +1,9 @@
 # WARNING - THIS FILE WAS GENERATED BY THE dps-gradle-spring-boot GRADLE PLUGIN
-#           AND ANY MANUAL CHANGES WILL BE OVERRIDDEN ON YOUR NEXT BUILD.
-#
-# To make general changes to the suppressions below, change the gradle plugin dps-gradle-spring-boot,
-# publish a new version and update to the new version in your gradle build script
-#
-# To stop the dps-gradle-spring-boot project from overwriting any project specific customisations here, remove the
-# warning at the top of this file.
+ #           AND ANY MANUAL CHANGES WILL BE OVERRIDDEN ON YOUR NEXT BUILD.
+ #
+ # To make general changes to the suppressions below, change the gradle plugin dps-gradle-spring-boot,
+ # publish a new version and update to the new version in your gradle build script
+ #
+ # To stop the dps-gradle-spring-boot project from overwriting any project specific customisations here, remove the
+ # warning at the top of this file.
 
-# suppress false positive for log4j.api - https://spring.io/blog/2021/12/10/log4j2-vulnerability-and-spring-boot
-CVE-2021-44228
```

Comment:
This conflicts with my PR, I'll rebase mine to resolve

Reply:
Sorry. The plugin update means this suppression disappears. Thanks.